.github/workflows/gradle.yml

```diff
@@ -102,6 +102,57 @@ jobs:
 #      - name: Setup upterm session
 #        uses: lhotari/action-upterm@v1
 
+  snapshot_docs:
```

Comment:
I have detected that Git Actions got stuck twice with this PR. When is the docs upload actually taking place? Is it executed in parallel to other jobs? Something is not right in the way GA behaves with this PR.

Reply:
I think I might have triggered a bug or race condition in GA by using force-pushes...